plot.py

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env python2.7
+
+import matplotlib
+matplotlib.use('Agg')
+matplotlib.rcParams['text.usetex'] = True
+matplotlib.rcParams['text.latex.preamble'] = [r'\usepackage[cm]{sfmath}']
+matplotlib.rcParams['font.family'] = 'sans-serif'
+matplotlib.rcParams['font.sans-serif'] = 'cm'
+import matplotlib.pyplot as plt
+import numpy as np
+from numpy.polynomial.polynomial import polyfit
+import os
+
+sort = []
```

Comment:
`sort` is a weird name for a list that spends a lot of its time being unsorted ;)